plugins/org.jkiss.dbeaver.tasks.native.ui/src/org/jkiss/dbeaver/tasks/ui/nativetool/AbstractNativeToolWizardPage.java

```diff
@@ -76,6 +77,9 @@ protected void updateState() {
         setPageComplete(true);
     }
 
+    protected void updateTableCheckedStatus(Table table, boolean check) {
+    }
+
```

Comment:
I'd rather refactor `updateState` to accept two additional parameters instead of creating a separate method.

Reply:
In my opinion, here refactoring will be overwhelmed. I need arguments from the method, but I do not want to throw them through the "updateState" method. It is used in 8 classes, but I'm interested in only two.

Reply:
Introduction of a new method creates a mess. If it is used in 8 classes, so what? They may eventually need such functionality. Instead of overloading another method, they would use these parameters instead.

Reply:
In 8 classes and in 22 places. I continue to assume that this is too big refactoring for such a small case.

Reply:
I'd refactor it further to avoid having `checkedObjects` in every inheritor. Move it to base class, add generics here and there. Don't you want to make our codebase cleaner?

At least consider adding `@NotNull` annotations to parameters in the introduced method.